setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="Open edX",
-    version="0.4",
+    version="0.4.1",
```

Comment:
I've been told that we should just make major version number updates, so this would be `0.5`.